__tests__/app.test.js

```diff
@@ -104,6 +104,55 @@ describe("/api/articles/:article_id/comments", () => {
         expect(body.comments).toEqual([]);
       });
   });
+
+  it("POST:201 responds with the correct comment object", () => {
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send({
+        username: "lurker",
+        body: "This is the greatest comment.  EVER.",
+      })
+      .set("Accept", "application/json")
+      .expect("Content-Type", /json/)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toMatchObject({
```

Comment:
It might be worth asserting that the comment_id comes back as well. It will _potentially_ be useful in front end.

Reply:
Cool - I've added an expect statement for that